src/main/play-doc/operation/Install.md

```diff
@@ -86,29 +91,77 @@ A typical response contains the current members of the cluster (shown here as ju
 }
 ```
 
-The IP addresses in the response indicate that ConductR is listening to the `localhost` address. To be able to form an inter-machine cluster, ConductR must be configured to listen to the machine's host interface. This can be enabled adding a property declaration for `CONDUCTR_IP` to the start command as follows:
+The IP addresses in the response indicate that ConductR Core is listening to the `localhost` address. To be able to form an inter-machine cluster, ConductR Core must be configured to listen to the machine's host interface. This can be enabled adding a property declaration for `CONDUCTR_IP` to the start command as follows:
 
-``` bash
-[172.17.0.1]$ echo -DCONDUCTR_IP=$(hostname) | sudo tee -a /usr/share/conductr/conf/application.ini
+```bash
+[172.17.0.1]$ echo -DCONDUCTR_IP=$(hostname) | sudo tee -a /usr/share/conductr/conf/conductr.ini
 [172.17.0.1]$ sudo service conductr restart
 ```
 
 Check for the cluster information once again, but now use the host address of the machine.
 
-``` bash
+```bash
 [172.17.0.1]$ curl -s $(hostname):9005/members | python3 -m json.tool
 ```
 
+The ConductR Core service runs under the `conductr` user along with the `conductr` group. Its pid file is written to: `/var/run/conductr/running.pid` and its install location is `/usr/share/conductr`.
+
+### Installing ConductR Agent on the first machine
+
+The tutorial also assumes that you have obtained the `conductr-agent_%PLAY_VERSION%_all.deb` Debian or `conductr-agent_%PLAY_VERSION%-1.noarch.rpm` Rpm package.
+
+Install ConductR Agent as any other Debian or Rpm package.
+
+```bash
+[172.17.0.1]$ sudo dpkg -i conductr-agent_%PLAY_VERSION%_all.deb
+```
+or
+
+```bash
+[172.17.0.1]$ sudo yum install conductr-agent-%PLAY_VERSION%-1.noarch.rpm
+```
+
+ConductR Agent is automatically registered as a service and started.
+
+ConductR Agent needs to be connected to a ConductR core node in order for ConductR to run any application process. To establish this connection, configure ConductR Agent as such:
+
+```bash
+[172.17.0.1]$ echo -Dconductr.agent.ip=$(hostname) | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
+[172.17.0.1]$ echo --core-node $(hostname):9004 | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
+```
+
+By default ConductR Agent and ConductR Core are using `/tmp` as its working directory. Since both ConductR Agent and ConductR Core are installed in the same machine, move the ConductR Agent working directory as such:
+
+```bash
+[172.17.0.1]$ echo -Dconductr.agent.storage-dir=/tmp/conductr-agent/bundles | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
```

Comment:
Why don't we just change the agent's default setting so that we then don't have to mention it?